### DISCUSSION TITLE
v1.5.1-beta

### DISCUSSION THREAD

**Discussion by edevsh** [shahmal1yev/blueskysdk#44] [Announcements]
## What's Changed

* [BSS-99] Fix deprecated method warning suppression in SDK components by @shahmal1yev in https://github.com/shahmal1yev/blueskysdk/pull/42

**Full Changelog**: https://github.com/shahmal1yev/blueskysdk/compare/v1.5.0-beta1...v1.5.1-beta

<hr /><em>This discussion was created from the release <a href='https://github.com/shahmal1yev/blueskysdk/releases/tag/v1.5.1-beta'>v1.5.1-beta</a>.</em>

**Comment by dipenpro** [comment]
Is there any option to post video?

**Reply by edevsh** [reply]
Yes, of course! I'm currently working on new PRs to enhance the package... :)

**Reply by dipenpro** [reply]
Could you please provide any tentative time for same?

**Reply by edevsh** [reply]
I think I will release a new version by the end of the day.

**Reply by edevsh** [reply]
I had worked on it a bit the day I saw your question, and although the result was successful, I think it would be better if I go over the changes once more: [Example post](https://bsky.app/profile/shahmal1yevv.bsky.social/post/3lgsatfnekh22)

**Comment by edevsh** [comment]
(@dipenpro) It was implemented. If you'd like to see an example, please refer to the following link:  
[Video Upload Example](https://github.com/shahmal1yev/blueskysdk/wiki/Examples#video-upload)

I'm closing the discussion for now. If you encounter any issues, feel free to create a new issue or report it here at any time.
